inc/validation.php

```diff
@@ -84,7 +84,8 @@ function ppom_sanitize_array_data( $array ) {
 		if ( is_array( $value ) ) {
 			$value = ppom_sanitize_array_data( $value );
 		} else {
-			if ( in_array( $key, ppom_fields_with_html() ) ) {
+			$value = preg_replace( '@<(script|style)[^>]*?>.*?</\\1>@si', '', $value );
+			if ( in_array( $key, ppom_fields_with_html(), true ) ) {
```

Comment:
I would like to suggest sanitization on all key&values of $array the $by Wordpress sanitize functions instead of `preg_replace` call.

Reply:
@mskapusuz Thank you for the suggestion, I've stripped all HTML tags using the WP function, please check with the latest commit.